bindings/matrix-sdk-ffi/src/authentication_service.rs

```diff
@@ -113,6 +117,59 @@ impl AuthenticationService {
         }
     }
 
+    /// Login to the current homeserver using an existing access token that was
+    /// issued by an authentication server.
+    pub fn login_access_token(&self, token: String) -> Result<Arc<Client>, AuthenticationError> {
```

Comment:
I would probably name this `login_with_access_token()` or perhaps `login_with_token()`. Not really needed though.

Reply:
Ah right, that one is already taken. Good suggestions, thanks.

Reply:
Sounds good to me.